test/sql/test_substrait_tpcds.test

```diff
@@ -0,0 +1,409 @@
+# name: test/sql/test_substrait_tpcds.test
+# description: Test get_substrait with TPC-DS queries
+# group: [sql]
+
+require substrait
+
+require tpcds
+
+statement ok
+PRAGMA enable_verification
+
+statement ok
+CALL dsdgen(sf=0.1)
+
+#Q 1 (DELIM_JOIN)
+#statement ok
+#CALL get_substrait('WITH customer_total_return AS (SELECT sr_customer_sk AS ctr_customer_sk, sr_store_sk AS ctr_store_sk, Sum(sr_return_amt) AS ctr_total_return FROM store_returns, date_dim WHERE sr_returned_date_sk = d_date_sk AND d_year = 2001 GROUP BY sr_customer_sk, sr_store_sk) SELECT c_customer_id FROM customer_total_return ctr1, store, customer WHERE ctr1.ctr_total_return > (SELECT Avg(ctr_total_return) * 1.2 FROM customer_total_return ctr2 WHERE ctr1.ctr_store_sk = ctr2.ctr_store_sk) AND s_store_sk = ctr1.ctr_store_sk AND s_state = ''TN'' AND ctr1.ctr_customer_sk = c_customer_sk ORDER BY c_customer_id LIMIT 100; ')
```

Comment:
Instead of commenting out a query, could we have statement error and the error message it generates?

something like:

```
statement error
CALL get_substrait('WITH customer_total_return AS (SELECT sr_customer_sk AS ctr_customer_sk, sr_store_sk AS ctr_store_sk, Sum(sr_return_amt) AS ctr_total_return FROM store_returns, date_dim WHERE sr_returned_date_sk = d_date_sk AND d_year = 2001 GROUP BY sr_customer_sk, sr_store_sk) SELECT c_customer_id FROM customer_total_return ctr1, store, customer WHERE ctr1.ctr_total_return > (SELECT Avg(ctr_total_return) * 1.2 FROM customer_total_return ctr2 WHERE ctr1.ctr_store_sk = ctr2.ctr_store_sk) AND s_store_sk = ctr1.ctr_store_sk AND s_state = ''TN'' AND ctr1.ctr_customer_sk = c_customer_sk ORDER BY c_customer_id LIMIT 100; ')
----
Not Implemented Exception: DELIM JOIN
```

That won't work if the query crashes, if any do we can leave it commented out for now!